### ISSUE TITLE
wb 3.14

### ISSUE THREAD

**Issue by uigiflip** [henrikstengaard/hstwb-installer#22]
is support for 3.14 going to be added?

**Comment by henrikstengaard** [comment]
Yes, i’m currently adding support for Amiga OS 3.1.4 in v1.3.0-beta1 and hope to have this beta version released before new year

**Comment by uigiflip** [comment]
https://drive.google.com/file/d/1WFr-XGCtwgUcL_dFGXBBpmTSJUjb1ktN/view

new classic wb with 3.14 support

**Comment by henrikstengaard** [comment]
Nice, it would be great to extract the changed files to make a patch that can make the existing classic wb package support both amiga os 3.1 and 3.1.4

**Comment by uigiflip** [comment]
not sure theres a new hdf with new scripts

**Comment by uigiflip** [comment]
a new update again, this fixes issue with cloanto 3.1 2014 release

https://drive.google.com/file/d/1RIBLd3j9OVla5ngHSpj8S2ccz56QIvtP/view

where would miss files eg workbench.ibrary and fail to recognise install disc

**Comment by Renaud-Pagin** [comment]
I have a strange Issue, i took last trunk "3.1.4" i copied ADF of Wb 3.1.4 as stated in the detect script

amiga-os-314-[storage/workbench/fonts/install/locale/module-a1200].adf

When i want to select it , it is never detected i have always 3.1 cloanto / custom detected & valid.

I checked MD5 as well , they are valid.

Do i miss something somewhere ?

**Comment by henrikstengaard** [comment]
@uigiflip I must admit I haven't been focusing on adding support for Amiga OS 3.X mostly because I don't own it and not sure how popular it is compared to Amiga OS 3.1 or 3.9.

**Comment by uigiflip** [comment]
Well 3.x rom seems more suitable for 3.9. Always had issues if try do fancy
3.1 install with it.

**Comment by uigiflip** [comment]
3.x works with

3.9, classic workbench 3.9 and real amikit works with it on a1200.

**Comment by uigiflip** [comment]
3.14 wb support now been added for classic wb 68k https://drive.google.com/file/d/1ZrjjE0fzfe8eRKywsy6v6BbSPV3WrcIw/view

**Comment by uigiflip** [comment]
v3.11 of classic wb full https://drive.google.com/file/d/1RIBLd3j9OVla5ngHSpj8S2ccz56QIvtP/view

3.14 support

**Comment by henrikstengaard** [comment]
@uigiflip Thanks, it great with more support for Amiga OS 3.1.4.

Yes, HstWB Installer v1.3.0-BETA1 is almost ready for release and I plan to release it tomorrow as 4GB, 8GB and 16GB self install images. HstWB Installer itself will also be released, but won't have support for running install mode. Currently it can only build self install images with Amiga OS 3.1.4 support.

**Comment by uigiflip** [comment]
checked today, can not find HstWB Installer v1.3.0-BETA1, nor the self install betas

**Comment by henrikstengaard** [comment]
Right, I had to postpone the release until this evening due to last minute taking longer than expected. I will update this issue once the release is available for download

**Comment by uigiflip** [comment]
also new 68k classic wb image, https://drive.google.com/open?id=1peJHzAX1mJdVcaKUBHEHngWhZLiAetuy

**Comment by henrikstengaard** [comment]
Thanks, HstWB Installer v1.3.0-BETA1 is released for download at github releases and at https://hstwb.firstrealize.com.

**Comment by uigiflip** [comment]
tested 3.1.4wb with best wb so far, seems working, couldnt believe wb 3.14 installed in like 10 secs. Is new classic wb integrated or is it still old v28 or new 3.1.4 classic wb installs?

**Comment by antob** [comment]
Hello! I tried the HstWB installer v1.3.0-BETA1 tonight. Installation went smooth. Only issue I ran into was that I had to manually replace the token `[$AmigaOsHarddriveIndex]` and `[$AmigaOsDir]` in `hstwb-installer.fs-uae` (after running `hstwb_image_setup.sh`, before starting FS-UAE).

This is a fantastic project, thank you so much for your work! Much appreciated!

**Comment by henrikstengaard** [comment]
@antob Thanls, I’m glad to hear that it almost went smooth using latest beta release and fs-uae. I will take your findings and correct them for next beta release.

The reason for the requester being shown for loading Libs/Workbench.library is due to Amiga OS 3.1.4 Kickstart rom doesn’t contain this library to make room for other improvements. Kickstart 3.1 and older Kickstart roms has this library included, so using those you wont see such requesters. You’re right that inserting Install 3.1.4 disk solves it and HstWB Installer should also remind users to eject inserted the disk afterwards to prevent it being booted from later.

**Comment by henrikstengaard** [comment]
The issue with [$AmigaOsHarddriveIndex] placeholder not being replaced properly by hstwb_image_setup scripts has been fixed in HstWB Installer v1.3.0-BETA2. Self install has also been changed to avoid requesters for "Libs:workbench.library" and "Libs:icon.library".

HstWB Installer v1.3.0-BETA2 has full support for AMiga OS 3.1.4 installation.

**Comment by henrikstengaard** [comment]
@uigiflip I'm now focusing on getting your Amiga OS 3.1.4 updates for ClassicWB integrated in the packages for final release of HstWB Installer v1.3.0.

**Comment by henrikstengaard** [comment]
HstWB Installer v1.3.0 is released with full support for AmigaOS 3.1.4
